Makefile

```diff
@@ -25,12 +27,24 @@ dev:
 .PHONY: test
 test:
 	go test -v ./api
-	go test -v ./geography
+	go test -v ./census_acs5
+  go test -v ./goegraphy
```

Comment:
indentation is important for makefiles

Reply:
Yeah I’ll fix that when I get the chance, got messed up when editing in github merge conflict editor.